lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java

```diff
@@ -110,30 +111,24 @@ private static Configuration createClusteredLockCacheConfiguration(ClusteredLock
       new Thread(() -> {
          try {
             Cache<? extends ClusteredLockKey, ClusteredLockValue> locksCache = cacheManager.getCache(CLUSTERED_LOCK_CACHE_NAME);
-            future.complete(
-                  new CacheHolder(locksCache.getAdvancedCache()));
+            future.complete(new CacheHolder(locksCache.getAdvancedCache()));
          } catch (Throwable throwable) {
             future.completeExceptionally(throwable);
          }
       }).start();
       return future;
    }
 
-   private static void registerClusteredLockManager(GlobalComponentRegistry registry,
+   private static void registerClusteredLockManager(BasicComponentRegistry registry,
+                                                    GlobalConfiguration globalConfig,
                                                     CompletableFuture<CacheHolder> future,
                                                     ClusteredLockManagerConfiguration config) {
-      //noinspection SynchronizationOnLocalVariableOrMethodParameter
-      synchronized (registry) {
```

Comment:
Do know why this was synchronized before?

Reply:
For no particular reason. It was copied from counters years ago. Meanwhile counters no longer uses synchronization there (because it does not make sense).